docs/general.md

```diff
@@ -0,0 +1,43 @@
+# General
+For whom is this documentation intended? x y and z so we will cover ... for a reason.
+
+## Technologies used
+
+|Technology   	|Version   	|
+|---	|---	|
+|       |       |
+
+## Project structure
+
+```App/
+Controllers/
+    User/
+    Admin/
+
+Models/
+    User/
+    Seller/
+
+Views/
+    User/
+    Seller/
+```
+
+## Cheatsheets
+The following list facilitates a usefull set of commands to execute basic operations in this virtual environment.<br> This list serves as a summary of all the commands discussed in the documentation and can be used as guide throughout.  
+
+### General
+|Command   	|Effect   	|
+|---	|---	|
+|```pipenv shell```   	| Enter virtual environment   	|
+|```deactivate```   	| Leave virtual environment   	|
+
+### Sphinx documentation
```

Comment:
We spoke about having these as makefile commands. It's cool to have them here, but please also refer to the make command. Ideally, people would use the make commands, leaving us free to change them if need be.

Reply:
I've added them. I also noticed that this "general" table might get a little cluttered later on as there are a lot of commands that we might see as noteworthy. Perhaps it is either good to keep this as small as possible, have a way of showing the reading that some are important once or just splitting it to other tables. For now I've moved the entering and leaving virtual environment to the miscellaneous table.